tests/functional/api/test_member_roles.py

```diff
@@ -0,0 +1,18 @@
+"""
+GitLab API:
+https://docs.gitlab.com/ee/api/member_roles.html
+"""
+
+
+def test_instance_member_role(gl):
```

Comment:
We could add a marker here as it's EE-only. But not a big issue.

```suggestion
@pytest.mark.gitlab_premium
def test_instance_member_role(gl):
```